lib/bamboo/email.ex

```diff
@@ -68,11 +68,12 @@ defmodule Bamboo.Email do
   """
 
   @type address :: String.t | {String.t, String.t}
+  @type address_list :: nil | address | [address]
```

Comment:
You can implement the Bamboo.Formatter protocol for anything. Maps, structs, atoms, boo leans, anything that protocols allow you to implement. So any value you want could be formatted. We could say that it accepts a map or list of maps, but that wouldn't really be technically correct :S

I suppose we could just say that anything other than a map/struct would be weird to format with a protocol and just add a type for that but I'm not totally sure about that. 

Or maybe there a way to add a type for anything that implements a protocol? That would work best if possible. 

Sent from my iPhone

> On May 24, 2016, at 8:52 AM, Jason Draper <notifications@github.com> wrote:
> 
> In lib/bamboo/email.ex:
> 
> > @@ -68,11 +68,12 @@ defmodule Bamboo.Email do
> >    """
> >  
> >    @type address :: String.t | {String.t, String.t}
> > +  @type address_list :: nil | address | [address]
> Can you explain that more?
> 
> I don't like any if it can be avoided. That doesn't allow type checking anymore.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>